Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -1041,3 +1171,37 @@ void MenuBar::PatchHLEFunctions()
 {
   HLE::PatchFunctions();
 }
+
+void MenuBar::ClearCache()
+{
+  Core::RunAsCPUThread(JitInterface::ClearCache);
+}
+
+void MenuBar::LogInstructions()
+{
+  PPCTables::LogCompiledInstructions();
+}
+
+void MenuBar::SearchInstruction()
+{
+  bool good;
+  QString op = QInputDialog::getText(this, tr("Change instruction"), tr("New instruction:"),
```

Comment:
What does "Search" have to do with "Change" instruction?
Can ignore if thats the same Wx does (unless someone has a better idea on how to do this).